tests/core/test_bungiesearch.py

```diff
@@ -279,7 +279,10 @@ def test_time_indexing(self):
         except Exception as e:
```

Comment:
While working on this test, can we remove the exception handling here and let the testing framework catch it. Instead of an assertion fail, it's probably better for the test to raise the exception. Hence, this function would become:
```python
def test_time_indexing(self):
    update_index(Article.objects.all(), 'Article', start_date=datetime.strftime(datetime.now(), '%Y-%m-%d %H:%M'))
    update_index(NoUpdatedField.objects.all(), 'NoUpdatedField', end_date=datetime.strftime(datetime.now(), '%Y-%m-%d'))
```

What do you thinkg?